spec/resolver_spec.rb

```diff
@@ -182,7 +182,7 @@ def self.save!(path, name, index, requirements, resolved)
 Unable to satisfy the following requirements:
 
 - `json (>= 1.7.7)` required by `berkshelf (2.0.7)`
-- `json (>= 1.4.4, <= 1.7.7)` required by `chef (10.26)`
+- `json (<= 1.7.7, >= 1.4.4)` required by `chef (10.26)`
```

Comment:
why is this changing?

Reply:
I'll reword the commit messages with proper links to make it clear!

Reply:
@segiddins I added a proper explanation in https://github.com/CocoaPods/Molinillo/pull/139/commits/d42b00a38a140dd95ce2d739691762c026dd156e, and also added a few other changes to get CI green.